graphql.md

```diff
@@ -0,0 +1,177 @@
+# GraphQL
+
+Mutations:
+
+- [Name should begin with action type](#name-should-begin-with-action-type)
+- [Use an input object type for the argument](#use-an-input-object-type-for-the-argument)
+- [Return an object in the payload](#return-an-object-in-the-payload)
+- [Errors](#errors)
```

Comment:
If possible, it's best to organise the sections so each has a "instruction"-style title, like "Handle errors by doing x".